lib/llm/src/hub.rs

```diff
@@ -45,6 +45,7 @@ pub async fn from_hf(name: impl AsRef<Path>, ignore_weights: bool) -> anyhow::Re
     let api = ApiBuilder::new()
         .with_progress(true)
         .with_token(token)
+        .high()
```

Comment:
Description on the API is a little scary for devs: https://github.com/huggingface/hf-hub/blob/3c9391bb77a08c977cf20fe96a30861538dc56bd/src/api/tokio.rs#L256-L259
```
    /// This may cause issues on regular desktops as it will saturate
    /// CPUs by multiplexing the downloads.
```

Do you think this should be opt-in with an env var?

huggingface tooling itself supports some kind of opt-in faster download with an env var: https://huggingface.co/docs/huggingface_hub/guides/download#hftransfer

```
HF_HUB_ENABLE_HF_TRANSFER=1
```

Not saying we should use the same one necessarily, but maybe follow the pattern of opt-in.

Reply:
Ship it - can revisit if we see a reported issue around it